packages/playground/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "playground",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "backend": "^1.0.0",
+    "botframework-directlinejs": "0.9.18-master.3cece81",
```

Comment:
How to handle directlinejs version across all these package.json files?

Reply:
Packages that depends functionality from `botframework-directlinejs`, we will use `dependencies`:
- `bundle`: bundle Direct Line JS for end-user consumption

Packages that only requires typings from `botframework-directlinejs`, we will use `peerDependencies` and `devDependencies`:
- `backend`
- `component`: required for card actions types

We need to configure TypeScript to generate `*.d.ts* for both `backend` and `component`, based on the version of `botframework-directlinejs` they installed for dev. They will not bundle or depends on the actual `botframework-directlinejs`. So version are free as long as the object passed to `backend`/`component` fulfill the signature requirement.

What do you think?